examples/hardware_duplicator/src/hardware_duplicator_json.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+# This software was developed at the National Institute of Standards
+# and Technology by employees of the Federal Government in the course
+# of their official duties. Pursuant to title 17 Section 105 of the
+# United States Code this software is not subject to copyright
+# protection and is in the public domain. NIST assumes no
+# responsibility whatsoever for its use by other parties, and makes
+# no guarantees, expressed or implied, about its quality,
+# reliability, or any other characteristic.
+#
+# We would appreciate acknowledgement if the software is used.
+
+"""
+This script was copied from the Owl Trafficking scenario.
+"""
+
+__version__ = "0.1.0"
+
+import logging
+import json
+import os
+import sys
+
+_logger = logging.getLogger(os.path.basename(__file__))
+
+
+def main() -> None:
+    obj = None
+    with open(args.in_base_json, "r") as in_fh:
+        obj = json.load(in_fh)
+    assert isinstance(obj, dict)
+
+    _logger.debug("args.in_json = %r." % args.in_json)
+    for in_json in args.in_json:
+        _logger.debug("in_json = %r." % in_json)
+        with open(in_json, "r") as in_fh:
+            in_obj = json.load(in_fh)
+            assert isinstance(in_obj, list)
+
+            for item in in_obj:
+                obj["@graph"].append(item)
+
+    json.dump(obj, sys.stdout, indent=4)
+
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("in_base_json")
+    parser.add_argument("in_json", nargs="+")
+    args = parser.parse_args()
+    # logging.basicConfig(level=logging.DEBUG)
```

Comment:
Should this line be removed in favor of the one following?

Reply:
The most correct answer is to give each program the `--debug` flag, like the `case-utils` program sdo.  However, this would be better handled by consolidating all of these specializable-but-currently-just-copied scripts into one program like the CASE-Examples repo does.  I suggest this is not a merge impediment for this or #233.